/usr/local/lib/python2.6/dist-packages/Pylons-0.9.6.2-py2.6.egg/pylons/middleware.py:11: DeprecationWarning: The webhelpers.rails package is deprecated.
- Please begin migrating to the new helpers in webhelpers.html,
  webhelpers.text, webhelpers.number, etc.  
- Import url_for() directly from routes, and redirect_to() from
  pylons.controllers.util (if using Pylons) or from routes.
- All Javascript support has been deprecated.  You can write link_to_remote()
  yourself or use one of the third-party Javascript libraries.
  from webhelpers.rails.asset_tag import javascript_path
/home/szimpatikus/reddit/r2/r2/lib/translation.py:32: DeprecationWarning: the md5 module is deprecated; use hashlib instead
  from md5 import md5
/home/szimpatikus/reddit/r2/r2/lib/manager/tp_manager.py:22: DeprecationWarning: the sha module is deprecated; use the hashlib module instead
  import pylons, sha
Traceback (most recent call last):
  File "/usr/local/bin/paster", line 9, in <module>
    load_entry_point('PasteScript==1.7.3', 'console_scripts', 'paster')()
  File "/usr/local/lib/python2.6/dist-packages/PasteScript-1.7.3-py2.6.egg/paste/script/command.py", line 84, in run
    invoke(command, command_name, options, args[1:])
  File "/usr/local/lib/python2.6/dist-packages/PasteScript-1.7.3-py2.6.egg/paste/script/command.py", line 123, in invoke
    exit_code = runner.run(args)
  File "/usr/local/lib/python2.6/dist-packages/PasteScript-1.7.3-py2.6.egg/paste/script/command.py", line 218, in run
    result = self.command()
  File "/home/szimpatikus/reddit/r2/r2/commands.py", line 63, in command
    load_environment(conf.global_conf, conf.local_conf)
  File "/home/szimpatikus/reddit/r2/r2/config/environment.py", line 54, in load_environment
    config['pylons.g'] = app_globals.Globals(global_conf, app_conf, paths)
  File "/home/szimpatikus/reddit/r2/r2/lib/app_globals.py", line 181, in __init__
    HardCache(self)),
  File "/home/szimpatikus/reddit/r2/r2/lib/cache.py", line 263, in __init__
    self.backend = HardCacheBackend(gc)
  File "/home/szimpatikus/reddit/r2/r2/lib/hardcachebackend.py", line 52, in __init__
    self.tdb.create_table(self.table, [ indstr ])
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_lite.py", line 47, in create_table
    if not t.bind.has_table(t.name):
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1271, in has_table
    return self.run_callable(lambda c: self.dialect.has_table(c, table_name, schema=schema))
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1191, in run_callable
    conn = self.contextual_connect()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/threadlocal.py", line 194, in contextual_connect
    return self.session.get_connection(**kwargs)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/threadlocal.py", line 20, in get_connection
    return self.engine.TLConnection(self, self.engine.pool.connect(), close_with_result=close_with_result)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 151, in connect
    agent = _ConnectionFairy(self)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 304, in __init__
    rec = self._connection_record = pool.get()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 161, in get
    return self.do_get()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 639, in do_get
    con = self.create_connection()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 122, in create_connection
    return _ConnectionRecord(self)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 198, in __init__
    self.connection = self.__connect()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 261, in __connect
    connection = self.__pool._creator()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/strategies.py", line 80, in connect
    raise exc.DBAPIError.instance(None, None, e)
sqlalchemy.exc.OperationalError: (OperationalError) could not connect to server: Connection refused
	Is the server running on host "127.0.0.1" and accepting
	TCP/IP connections on port 5432?
 None None
/usr/local/lib/python2.6/dist-packages/Pylons-0.9.6.2-py2.6.egg/pylons/middleware.py:11: DeprecationWarning: The webhelpers.rails package is deprecated.
- Please begin migrating to the new helpers in webhelpers.html,
  webhelpers.text, webhelpers.number, etc.  
- Import url_for() directly from routes, and redirect_to() from
  pylons.controllers.util (if using Pylons) or from routes.
- All Javascript support has been deprecated.  You can write link_to_remote()
  yourself or use one of the third-party Javascript libraries.
  from webhelpers.rails.asset_tag import javascript_path
/home/szimpatikus/reddit/r2/r2/lib/translation.py:32: DeprecationWarning: the md5 module is deprecated; use hashlib instead
  from md5 import md5
/home/szimpatikus/reddit/r2/r2/lib/manager/tp_manager.py:22: DeprecationWarning: the sha module is deprecated; use the hashlib module instead
  import pylons, sha
Traceback (most recent call last):
  File "/usr/local/bin/paster", line 9, in <module>
    load_entry_point('PasteScript==1.7.3', 'console_scripts', 'paster')()
  File "/usr/local/lib/python2.6/dist-packages/PasteScript-1.7.3-py2.6.egg/paste/script/command.py", line 84, in run
    invoke(command, command_name, options, args[1:])
  File "/usr/local/lib/python2.6/dist-packages/PasteScript-1.7.3-py2.6.egg/paste/script/command.py", line 123, in invoke
    exit_code = runner.run(args)
  File "/usr/local/lib/python2.6/dist-packages/PasteScript-1.7.3-py2.6.egg/paste/script/command.py", line 218, in run
    result = self.command()
  File "/home/szimpatikus/reddit/r2/r2/commands.py", line 63, in command
    load_environment(conf.global_conf, conf.local_conf)
  File "/home/szimpatikus/reddit/r2/r2/config/environment.py", line 54, in load_environment
    config['pylons.g'] = app_globals.Globals(global_conf, app_conf, paths)
  File "/home/szimpatikus/reddit/r2/r2/lib/app_globals.py", line 181, in __init__
    HardCache(self)),
  File "/home/szimpatikus/reddit/r2/r2/lib/cache.py", line 263, in __init__
    self.backend = HardCacheBackend(gc)
  File "/home/szimpatikus/reddit/r2/r2/lib/hardcachebackend.py", line 52, in __init__
    self.tdb.create_table(self.table, [ indstr ])
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_lite.py", line 47, in create_table
    if not t.bind.has_table(t.name):
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1271, in has_table
    return self.run_callable(lambda c: self.dialect.has_table(c, table_name, schema=schema))
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1191, in run_callable
    conn = self.contextual_connect()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/threadlocal.py", line 194, in contextual_connect
    return self.session.get_connection(**kwargs)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/threadlocal.py", line 20, in get_connection
    return self.engine.TLConnection(self, self.engine.pool.connect(), close_with_result=close_with_result)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 151, in connect
    agent = _ConnectionFairy(self)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 304, in __init__
    rec = self._connection_record = pool.get()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 161, in get
    return self.do_get()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 639, in do_get
    con = self.create_connection()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 122, in create_connection
    return _ConnectionRecord(self)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 198, in __init__
    self.connection = self.__connect()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/pool.py", line 261, in __connect
    connection = self.__pool._creator()
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/strategies.py", line 80, in connect
    raise exc.DBAPIError.instance(None, None, e)
sqlalchemy.exc.OperationalError: (OperationalError) could not connect to server: Connection refused
	Is the server running on host "127.0.0.1" and accepting
	TCP/IP connections on port 5432?
 None None
/usr/local/lib/python2.6/dist-packages/Pylons-0.9.6.2-py2.6.egg/pylons/middleware.py:11: DeprecationWarning: The webhelpers.rails package is deprecated.
- Please begin migrating to the new helpers in webhelpers.html,
  webhelpers.text, webhelpers.number, etc.  
- Import url_for() directly from routes, and redirect_to() from
  pylons.controllers.util (if using Pylons) or from routes.
- All Javascript support has been deprecated.  You can write link_to_remote()
  yourself or use one of the third-party Javascript libraries.
  from webhelpers.rails.asset_tag import javascript_path
/home/szimpatikus/reddit/r2/r2/lib/translation.py:32: DeprecationWarning: the md5 module is deprecated; use hashlib instead
  from md5 import md5
/home/szimpatikus/reddit/r2/r2/lib/manager/tp_manager.py:22: DeprecationWarning: the sha module is deprecated; use the hashlib module instead
  import pylons, sha
reddit app a.v01.lanten.hu:787 started d251ba7 at 2011-04-07 12:55:14.226982
/usr/bin/env: optipng: No such file or directory
Warning: g.media_domain == g.domain. This may give untrusted content access to user cookies
error connecting to amqp reddit @ localhost:5672
error connecting to amqp reddit @ localhost:5672
error connecting to amqp reddit @ localhost:5672
error connecting to amqp reddit @ localhost:5672
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
Error fetching t5_2c0
Traceback (most recent call last):
  File "<string>", line 110, in process_link
  File "/home/szimpatikus/reddit/r2/r2/lib/utils/utils.py", line 1167, in __call__
    result = self.function(*args)
  File "<string>", line 106, in _process_link
  File "<string>", line 94, in set_media
  File "<string>", line 75, in update_link
  File "/home/szimpatikus/reddit/r2/r2/lib/db/thing.py", line 241, in _commit
    self._set_data(self._type_id, self._id, **data_props)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 572, in set_thing_data
    return set_data(table, type_id, thing_id, **vals)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 511, in set_data
    keys = [x.key for x in s.execute().fetchall()]
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/sql/expression.py", line 1122, in execute
    return e._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1209, in _execute_clauseelement
    return connection._execute_clauseelement(elem, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 874, in _execute_clauseelement
    return self.__execute_context(context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 896, in __execute_context
    self._cursor_execute(context.cursor, context.statement, context.parameters[0], context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 950, in _cursor_execute
    self._handle_dbapi_exception(e, statement, parameters, cursor, context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 931, in _handle_dbapi_exception
    raise exc.DBAPIError.instance(statement, parameters, e, connection_invalidated=is_disconnect)
OperationalError: (OperationalError) terminating connection due to administrator command
server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
 'SELECT reddit_data_link.key \nFROM reddit_data_link \nWHERE reddit_data_link.thing_id = %(thing_id_1)s' {'thing_id_1': 3024L}

/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
Error fetching t5_2c6
Traceback (most recent call last):
  File "<string>", line 110, in process_link
  File "/home/szimpatikus/reddit/r2/r2/lib/utils/utils.py", line 1167, in __call__
    result = self.function(*args)
  File "<string>", line 106, in _process_link
  File "<string>", line 94, in set_media
  File "<string>", line 75, in update_link
  File "/home/szimpatikus/reddit/r2/r2/lib/db/thing.py", line 241, in _commit
    self._set_data(self._type_id, self._id, **data_props)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 572, in set_thing_data
    return set_data(table, type_id, thing_id, **vals)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 529, in set_data
    i.execute(*inserts)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/sql/expression.py", line 1122, in execute
    return e._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1209, in _execute_clauseelement
    return connection._execute_clauseelement(elem, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 874, in _execute_clauseelement
    return self.__execute_context(context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 894, in __execute_context
    self._cursor_executemany(context.cursor, context.statement, context.parameters, context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 960, in _cursor_executemany
    self._handle_dbapi_exception(e, statement, parameters, cursor, context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 931, in _handle_dbapi_exception
    raise exc.DBAPIError.instance(statement, parameters, e, connection_invalidated=is_disconnect)
DataError: (DataError) invalid byte sequence for encoding "UTF8": 0xe17461
HINT:  This error can also happen if the byte sequence does not match the encoding expected by the server, which is controlled by "client_encoding".
 'INSERT INTO reddit_data_link (thing_id, key, value, kind) VALUES (%(thing_id)s, %(key)s, %(value)s, %(kind)s)' [{'kind': 'bool', 'thing_id': 3030L, 'key': 'has_thumbnail', 'value': 't'}, {'kind': 'pickle', 'thing_id': 3030L, 'key': 'media_object', 'value': '(dp1\nS\'oembed\'\np2\n(dp3\nS\'provider_url\'\np4\nS\'http://www.youtube.com/\'\np5\nsS\'description\'\np6\nVCreated at Sapientia University, Tirgu Mures, Romania.\\u000aDirected by K\xe1tai Zolt\xe1n and T\xf3th L\xe1szl\xf3\\u000aIn collaboration with "Maros m\\u0171v\xe9szegy\xfcttes", Tirgu Mures, Romania.\\u000aChoreographer: F\xfczesi Albert\\u000aVideo: L\\u0151rinc Lajos, K\xf6rm\xf6cki Zolt\xe1n\\u000aSupported by "Sz\xfcl\\u0151f\xf6ld Alap" and evoline company.\np7\nsS\'title\'\np8\nVBubble-sort with Hungarian ("Cs\xe1ng\xf3") folk dance\np9\nsS\'url\'\np10\nS\'http://www.youtube.com/watch?v=lyZQPjUT5B4\'\np11\nsS\'type\'\np12\nS\'video\'\np13\nsS\'author_name\'\np14\nS\'AlgoRythmics\'\np15\nsS\'height\'\np16\nI363\nsS\'width\'\np17\nI600\nsS\'html\'\np18\nS\'<object width="600" height="363"><param name="movie" value="http://www.youtube.com/v/lyZQPjUT5B4?version=3"></param><param name="allowFullScreen" value="true"></param><param name="allowscriptaccess" value="always"></param><embed src="http://www.youtube.com/v/lyZQPjUT5B4?version=3" type="application/x-shockwave-flash" width="600" height="363" allowscriptaccess="always" allowfullscreen="true"></embed></object>\'\np19\nsS\'thumbnail_width\'\np20\nI480\nsS\'version\'\np21\nS\'1.0\'\np22\nsS\'provider_name\'\np23\nS\'YouTube\'\np24\nsS\'thumbnail_url\'\np25\nS\'http://i1.ytimg.com/vi/lyZQPjUT5B4/hqdefault.jpg\'\np26\nsS\'thumbnail_height\'\np27\nI360\nsS\'author_url\'\np28\nS\'http://www.youtube.com/user/AlgoRythmics\'\np29\nssS\'type\'\np30\nS\'youtube.com\'\np31\nsS\'deep\'\np32\nVhttp://miafene.hu/2011/04/08/magyar-nemzeti-buborekrendezes/\np33\ns.'}]

/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
Timed out on t5_2cc
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
Error fetching t5_2cj
Traceback (most recent call last):
  File "<string>", line 110, in process_link
  File "/home/szimpatikus/reddit/r2/r2/lib/utils/utils.py", line 1167, in __call__
    result = self.function(*args)
  File "<string>", line 106, in _process_link
  File "<string>", line 94, in set_media
  File "<string>", line 75, in update_link
  File "/home/szimpatikus/reddit/r2/r2/lib/db/thing.py", line 241, in _commit
    self._set_data(self._type_id, self._id, **data_props)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 572, in set_thing_data
    return set_data(table, type_id, thing_id, **vals)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 529, in set_data
    i.execute(*inserts)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/sql/expression.py", line 1122, in execute
    return e._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1209, in _execute_clauseelement
    return connection._execute_clauseelement(elem, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 874, in _execute_clauseelement
    return self.__execute_context(context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 894, in __execute_context
    self._cursor_executemany(context.cursor, context.statement, context.parameters, context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 960, in _cursor_executemany
    self._handle_dbapi_exception(e, statement, parameters, cursor, context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 931, in _handle_dbapi_exception
    raise exc.DBAPIError.instance(statement, parameters, e, connection_invalidated=is_disconnect)
DataError: (DataError) invalid byte sequence for encoding "UTF8": 0xf30a7037
HINT:  This error can also happen if the byte sequence does not match the encoding expected by the server, which is controlled by "client_encoding".
 'INSERT INTO reddit_data_link (thing_id, key, value, kind) VALUES (%(thing_id)s, %(key)s, %(value)s, %(kind)s)' [{'kind': 'bool', 'thing_id': 3043L, 'key': 'has_thumbnail', 'value': 't'}, {'kind': 'pickle', 'thing_id': 3043L, 'key': 'media_object', 'value': '(dp1\nS\'oembed\'\np2\n(dp3\nS\'provider_url\'\np4\nS\'http://www.youtube.com/\'\np5\nsS\'description\'\np6\nVhttp://www.dancsopeter.exofire.net\r\\u000ahttp://www.fakultativ.com\r\\u000a\r\\u000aCamera: Canon EOS 550D / T2i\r\\u000aLenses: Canon 18-135mm kit + Sigma 30mm 1.4\r\\u000aMusic: A.R. Rahman - Liberation in a Dream (127 Hours Soundtrack)\r\\u000aShot and edited by: Peter Dancs\xf3\np7\nsS\'title\'\np8\nS\'Budapest Timelapse 2011\'\np9\nsS\'url\'\np10\nS\'http://www.youtube.com/watch?v=uSG8pcCfV5M\'\np11\nsS\'type\'\np12\nS\'video\'\np13\nsS\'author_name\'\np14\nS\'Petyamester\'\np15\nsS\'height\'\np16\nI363\nsS\'width\'\np17\nI600\nsS\'html\'\np18\nS\'<object width="600" height="363"><param name="movie" value="http://www.youtube.com/v/uSG8pcCfV5M?version=3"></param><param name="allowFullScreen" value="true"></param><param name="allowscriptaccess" value="always"></param><embed src="http://www.youtube.com/v/uSG8pcCfV5M?version=3" type="application/x-shockwave-flash" width="600" height="363" allowscriptaccess="always" allowfullscreen="true"></embed></object>\'\np19\nsS\'thumbnail_width\'\np20\nI480\nsS\'version\'\np21\nS\'1.0\'\np22\nsS\'provider_name\'\np23\nS\'YouTube\'\np24\nsS\'thumbnail_url\'\np25\nS\'http://i2.ytimg.com/vi/uSG8pcCfV5M/hqdefault.jpg\'\np26\nsS\'thumbnail_height\'\np27\nI360\nsS\'author_url\'\np28\nS\'http://www.youtube.com/user/Petyamester\'\np29\nssS\'type\'\np30\nS\'youtube.com\'\np31\ns.'}]

/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/local/lib/python2.6/dist-packages/Pylons-0.9.6.2-py2.6.egg/pylons/middleware.py:11: DeprecationWarning: The webhelpers.rails package is deprecated.
- Please begin migrating to the new helpers in webhelpers.html,
  webhelpers.text, webhelpers.number, etc.  
- Import url_for() directly from routes, and redirect_to() from
  pylons.controllers.util (if using Pylons) or from routes.
- All Javascript support has been deprecated.  You can write link_to_remote()
  yourself or use one of the third-party Javascript libraries.
  from webhelpers.rails.asset_tag import javascript_path
/home/szimpatikus/reddit/r2/r2/lib/translation.py:32: DeprecationWarning: the md5 module is deprecated; use hashlib instead
  from md5 import md5
/home/szimpatikus/reddit/r2/r2/lib/manager/tp_manager.py:22: DeprecationWarning: the sha module is deprecated; use the hashlib module instead
  import pylons, sha
reddit app a.v01.lanten.hu:16846 started d251ba7 at 2011-04-11 08:48:26.688389
/usr/local/lib/python2.6/dist-packages/Pylons-0.9.6.2-py2.6.egg/pylons/middleware.py:11: DeprecationWarning: The webhelpers.rails package is deprecated.
- Please begin migrating to the new helpers in webhelpers.html,
  webhelpers.text, webhelpers.number, etc.  
- Import url_for() directly from routes, and redirect_to() from
  pylons.controllers.util (if using Pylons) or from routes.
- All Javascript support has been deprecated.  You can write link_to_remote()
  yourself or use one of the third-party Javascript libraries.
  from webhelpers.rails.asset_tag import javascript_path
/home/szimpatikus/reddit/r2/r2/lib/translation.py:32: DeprecationWarning: the md5 module is deprecated; use hashlib instead
  from md5 import md5
/home/szimpatikus/reddit/r2/r2/lib/manager/tp_manager.py:22: DeprecationWarning: the sha module is deprecated; use the hashlib module instead
  import pylons, sha
reddit app a.v01.lanten.hu:17336 started d251ba7 at 2011-04-11 08:49:48.165631
/usr/local/lib/python2.6/dist-packages/Pylons-0.9.6.2-py2.6.egg/pylons/middleware.py:11: DeprecationWarning: The webhelpers.rails package is deprecated.
- Please begin migrating to the new helpers in webhelpers.html,
  webhelpers.text, webhelpers.number, etc.  
- Import url_for() directly from routes, and redirect_to() from
  pylons.controllers.util (if using Pylons) or from routes.
- All Javascript support has been deprecated.  You can write link_to_remote()
  yourself or use one of the third-party Javascript libraries.
  from webhelpers.rails.asset_tag import javascript_path
/home/szimpatikus/reddit/r2/r2/lib/translation.py:32: DeprecationWarning: the md5 module is deprecated; use hashlib instead
  from md5 import md5
/home/szimpatikus/reddit/r2/r2/lib/manager/tp_manager.py:22: DeprecationWarning: the sha module is deprecated; use the hashlib module instead
  import pylons, sha
reddit app a.v01.lanten.hu:28598 started d251ba7 at 2011-04-11 09:54:32.940000
/usr/bin/env: optipng: No such file or directory
Warning: g.media_domain == g.domain. This may give untrusted content access to user cookies
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
Error fetching t5_2d1
Traceback (most recent call last):
  File "<string>", line 110, in process_link
  File "/home/szimpatikus/reddit/r2/r2/lib/utils/utils.py", line 1167, in __call__
    result = self.function(*args)
  File "<string>", line 106, in _process_link
  File "<string>", line 94, in set_media
  File "<string>", line 75, in update_link
  File "/home/szimpatikus/reddit/r2/r2/lib/db/thing.py", line 241, in _commit
    self._set_data(self._type_id, self._id, **data_props)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 572, in set_thing_data
    return set_data(table, type_id, thing_id, **vals)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 529, in set_data
    i.execute(*inserts)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/sql/expression.py", line 1122, in execute
    return e._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1209, in _execute_clauseelement
    return connection._execute_clauseelement(elem, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 874, in _execute_clauseelement
    return self.__execute_context(context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 894, in __execute_context
    self._cursor_executemany(context.cursor, context.statement, context.parameters, context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 960, in _cursor_executemany
    self._handle_dbapi_exception(e, statement, parameters, cursor, context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 931, in _handle_dbapi_exception
    raise exc.DBAPIError.instance(statement, parameters, e, connection_invalidated=is_disconnect)
DataError: (DataError) invalid byte sequence for encoding "UTF8": 0xf6726373
HINT:  This error can also happen if the byte sequence does not match the encoding expected by the server, which is controlled by "client_encoding".
 'INSERT INTO reddit_data_link (thing_id, key, value, kind) VALUES (%(thing_id)s, %(key)s, %(value)s, %(kind)s)' [{'kind': 'bool', 'thing_id': 3061L, 'key': 'has_thumbnail', 'value': 't'}, {'kind': 'pickle', 'thing_id': 3061L, 'key': 'media_object', 'value': '(dp1\nS\'oembed\'\np2\n(dp3\nS\'provider_url\'\np4\nS\'http://maps.google.com\'\np5\nsS\'description\'\np6\nS\'Find local businesses, view maps and get driving directions in Google Maps\'\np7\nsS\'title\'\np8\nVPesti S\xf6rcsarnok, Budapest, V\xe1mh\xe1z k\xf6r\xfat 16., Magyarorsz\xe1g - Google Maps\np9\nsS\'type\'\np10\nS\'rich\'\np11\nsS\'thumbnail_width\'\np12\nI256\nsS\'height\'\np13\nI350\nsS\'width\'\np14\nI425\nsS\'html\'\np15\nS\'<iframe width="425" height="350" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="http://maps.google.com/maps?f=q&source=s_q&hl=en&q=Pesti+S%C3%B6rcsarnok%2C+Budapest%2C+V%C3%A1mh%C3%A1z+k%C3%B6r%C3%BAt+16.%2C+Magyarorsz%C3%A1g&sll=47.488151%2C19.059382&sspn=0.008047%2C0.019248&ie=UTF8&hq=Pesti+S%C3%B6rcsarnok%2C&hnear=1056+Budapest%2C+V.+ker%C3%BClet%2C+V%C3%A1mh%C3%A1z+k%C3%B6r%C3%BAt+16%2C+Hungary&ll=47.488723%2C19.060051&spn=0.002717%2C0.004812&t=h&z=18&output=embed"></iframe>\'\np16\nsS\'version\'\np17\nS\'1.0\'\np18\nsS\'provider_name\'\np19\nS\'Google Maps\'\np20\nsS\'thumbnail_url\'\np21\nS\'http://mt1.google.com/vt/lyrs=h@149&hl=en&x=144951&s=&y=91679&z=18&s=Gali\'\np22\nsS\'thumbnail_height\'\np23\nI256\nssS\'type\'\np24\nS\'google.com\'\np25\ns.'}]

oEmbed call (http://api.embed.ly/v1/api/oembed?url=http%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DRDGlMGqmFQc%22&maxwidth=600&format=json) failed to return content for http://www.youtube.com/watch?v=RDGlMGqmFQc"
scraper_q: 1 item 
Error fetching t5_2d2
Traceback (most recent call last):
  File "<string>", line 110, in process_link
  File "/home/szimpatikus/reddit/r2/r2/lib/utils/utils.py", line 1167, in __call__
    result = self.function(*args)
  File "<string>", line 106, in _process_link
  File "<string>", line 89, in set_media
  File "/home/szimpatikus/reddit/r2/r2/lib/scraper.py", line 255, in media_object
    found = ds.find_media_object(self)
  File "/home/szimpatikus/reddit/r2/r2/lib/scraper.py", line 1331, in find_media_object
    mo['deep'] = scraper.url
TypeError: 'NoneType' object does not support item assignment

/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/local/lib/python2.6/dist-packages/Pylons-0.9.6.2-py2.6.egg/pylons/middleware.py:11: DeprecationWarning: The webhelpers.rails package is deprecated.
- Please begin migrating to the new helpers in webhelpers.html,
  webhelpers.text, webhelpers.number, etc.  
- Import url_for() directly from routes, and redirect_to() from
  pylons.controllers.util (if using Pylons) or from routes.
- All Javascript support has been deprecated.  You can write link_to_remote()
  yourself or use one of the third-party Javascript libraries.
  from webhelpers.rails.asset_tag import javascript_path
/home/szimpatikus/reddit/r2/r2/lib/translation.py:32: DeprecationWarning: the md5 module is deprecated; use hashlib instead
  from md5 import md5
/home/szimpatikus/reddit/r2/r2/lib/manager/tp_manager.py:22: DeprecationWarning: the sha module is deprecated; use the hashlib module instead
  import pylons, sha
reddit app a.v01.lanten.hu:10766 started d251ba7 at 2011-04-11 11:16:43.290732
/usr/bin/env: optipng: No such file or directory
Warning: g.media_domain == g.domain. This may give untrusted content access to user cookies
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
oEmbed call (http://api.embed.ly/v1/api/oembed?url=http%3A%2F%2Fwww.facebook.com%2Fevent.php%3Feid%3D162486820476048&maxwidth=600&format=json) failed to return content for http://www.facebook.com/event.php?eid=162486820476048
oEmbed call (http://api.embed.ly/v1/api/oembed?url=http%3A%2F%2Fwww.facebook.com%2Fevent.php%3Feid%3D162486820476048&maxwidth=600&format=json) failed to return content for http://www.facebook.com/event.php?eid=162486820476048
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
oEmbed call (http://api.embed.ly/v1/api/oembed?url=http%3A%2F%2Fngm.nationalgeographic.com%2F2011%2F01%2Flargest-cave%2Fpeter-photography&maxwidth=600&format=json) failed to return content for http://ngm.nationalgeographic.com/2011/01/largest-cave/peter-photography
oEmbed call (http://api.embed.ly/v1/api/oembed?url=http%3A%2F%2Fngm.nationalgeographic.com%2F2011%2F01%2Flargest-cave%2Fpeter-photography&maxwidth=600&format=json) failed to return content for http://ngm.nationalgeographic.com/2011/01/largest-cave/peter-photography
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
Error fetching t5_2dp
Traceback (most recent call last):
  File "<string>", line 110, in process_link
  File "/home/szimpatikus/reddit/r2/r2/lib/utils/utils.py", line 1167, in __call__
    result = self.function(*args)
  File "<string>", line 106, in _process_link
  File "<string>", line 94, in set_media
  File "<string>", line 75, in update_link
  File "/home/szimpatikus/reddit/r2/r2/lib/db/thing.py", line 241, in _commit
    self._set_data(self._type_id, self._id, **data_props)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 572, in set_thing_data
    return set_data(table, type_id, thing_id, **vals)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 529, in set_data
    i.execute(*inserts)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/sql/expression.py", line 1122, in execute
    return e._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1209, in _execute_clauseelement
    return connection._execute_clauseelement(elem, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 874, in _execute_clauseelement
    return self.__execute_context(context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 894, in __execute_context
    self._cursor_executemany(context.cursor, context.statement, context.parameters, context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 960, in _cursor_executemany
    self._handle_dbapi_exception(e, statement, parameters, cursor, context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 931, in _handle_dbapi_exception
    raise exc.DBAPIError.instance(statement, parameters, e, connection_invalidated=is_disconnect)
DataError: (DataError) invalid byte sequence for encoding "UTF8": 0xe97420
HINT:  This error can also happen if the byte sequence does not match the encoding expected by the server, which is controlled by "client_encoding".
 'INSERT INTO reddit_data_link (thing_id, key, value, kind) VALUES (%(thing_id)s, %(key)s, %(value)s, %(kind)s)' [{'kind': 'bool', 'thing_id': 3085L, 'key': 'has_thumbnail', 'value': 't'}, {'kind': 'pickle', 'thing_id': 3085L, 'key': 'media_object', 'value': '(dp1\nS\'oembed\'\np2\n(dp3\nS\'provider_url\'\np4\nS\'http://www.youtube.com/\'\np5\nsS\'description\'\np6\nVA Spektrum Tv k\xe9t digit\xe1lis riportere, Cseke Eszter \xe9s S. Tak\xe1cs Andr\xe1s a forrong\xf3 arab vil\xe1gba utazik, hogy kider\xedtse: merre billen majd a k\xf6zel-keleti m\xe9rleg nyelve, \xe9s mit akarnak kezdeni az egyiptomiak a frissen szerzett szabads\xe1ggal. Piramisok, ahonnan elt\\u0171ntek a turist\xe1k a felkel\xe9s miatt, t\xf6bb mint 10 milli\xf3 kereszt\xe9ny, aki att\xf3l f\xe9l, hogy az egykori terrorszervezet, a Muzulm\xe1n Testv\xe9ris\xe9g ker\xfcl majd hatalomra \xe9s emberek, akiken nem seg\xedtett a forradalom: tov\xe1bbra is kript\xe1kban laknak Kair\xf3 sz\xe9l\xe9n. A holtak v\xe1ros\xe1t\xf3l a Tahrir t\xe9rig -- On The Spot: Egyiptom \xe1prilisban csak a Spektrum m\\u0171sor\xe1n! Ad\xe1sok: onthespot.spektrumtv.hu\np7\nsS\'title\'\np8\nVOTS: Egyiptom, Holtak V\xe1rosa\np9\nsS\'url\'\np10\nS\'http://www.youtube.com/watch?v=alv6ByYWvJw\'\np11\nsS\'type\'\np12\nS\'video\'\np13\nsS\'author_name\'\np14\nS\'onthespotvideo\'\np15\nsS\'height\'\np16\nI363\nsS\'width\'\np17\nI600\nsS\'html\'\np18\nS\'<object width="600" height="363"><param name="movie" value="http://www.youtube.com/v/alv6ByYWvJw?version=3"></param><param name="allowFullScreen" value="true"></param><param name="allowscriptaccess" value="always"></param><embed src="http://www.youtube.com/v/alv6ByYWvJw?version=3" type="application/x-shockwave-flash" width="600" height="363" allowscriptaccess="always" allowfullscreen="true"></embed></object>\'\np19\nsS\'thumbnail_width\'\np20\nI480\nsS\'version\'\np21\nS\'1.0\'\np22\nsS\'provider_name\'\np23\nS\'YouTube\'\np24\nsS\'thumbnail_url\'\np25\nS\'http://i2.ytimg.com/vi/alv6ByYWvJw/hqdefault.jpg\'\np26\nsS\'thumbnail_height\'\np27\nI360\nsS\'author_url\'\np28\nS\'http://www.youtube.com/user/onthespotvideo\'\np29\nssS\'type\'\np30\nS\'youtube.com\'\np31\nsS\'deep\'\np32\nVhttp://hvg.hu/vilag/20110413_holtak_varosa_egyiptom\np33\ns.'}]

scraper_q: 1 item 
oEmbed call (http://api.embed.ly/v1/api/oembed?url=http%3A%2F%2Fimgur.com%2Fa%2FM36uI&maxwidth=600&format=json) failed to return content for http://imgur.com/a/M36uI
oEmbed call (http://api.embed.ly/v1/api/oembed?url=http%3A%2F%2Fimgur.com%2Fa%2FM36uI&maxwidth=600&format=json) failed to return content for http://imgur.com/a/M36uI
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
scraper_q: 1 item 
oEmbed call (http://api.embed.ly/v1/api/oembed?url=http%3A%2F%2Fwww.facebook.com%2Fevent.php%3Feid%3D117433698332297&maxwidth=600&format=json) failed to return content for http://www.facebook.com/event.php?eid=117433698332297
oEmbed call (http://api.embed.ly/v1/api/oembed?url=http%3A%2F%2Fwww.facebook.com%2Fevent.php%3Feid%3D117433698332297&maxwidth=600&format=json) failed to return content for http://www.facebook.com/event.php?eid=117433698332297
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
Timed out on t5_2eh
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
oEmbed call (http://api.embed.ly/v1/api/oembed?url=https%3A%2F%2Fwww.facebook.com%2Fvideo%2Fvideo.php%3Fv%3D1686844565157&maxwidth=600&format=json) failed to return content for https://www.facebook.com/video/video.php?v=1686844565157
oEmbed call (http://api.embed.ly/v1/api/oembed?url=https%3A%2F%2Fwww.facebook.com%2Fvideo%2Fvideo.php%3Fv%3D1686844565157&maxwidth=600&format=json) failed to return content for https://www.facebook.com/video/video.php?v=1686844565157
scraper_q: 1 item 
scraper_q: 1 item 
Timed out on t5_2fb
scraper_q: 1 item 
Timed out on t5_2fc
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
Error fetching t5_2fd
Traceback (most recent call last):
  File "<string>", line 110, in process_link
  File "/home/szimpatikus/reddit/r2/r2/lib/utils/utils.py", line 1167, in __call__
    result = self.function(*args)
  File "<string>", line 106, in _process_link
  File "<string>", line 94, in set_media
  File "<string>", line 75, in update_link
  File "/home/szimpatikus/reddit/r2/r2/lib/db/thing.py", line 241, in _commit
    self._set_data(self._type_id, self._id, **data_props)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 572, in set_thing_data
    return set_data(table, type_id, thing_id, **vals)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 511, in set_data
    keys = [x.key for x in s.execute().fetchall()]
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/sql/expression.py", line 1122, in execute
    return e._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1209, in _execute_clauseelement
    return connection._execute_clauseelement(elem, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 874, in _execute_clauseelement
    return self.__execute_context(context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 896, in __execute_context
    self._cursor_execute(context.cursor, context.statement, context.parameters[0], context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 948, in _cursor_execute
    self.dialect.do_execute(cursor, statement, parameters, context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/default.py", line 122, in do_execute
    cursor.execute(statement, parameters)
SystemError: null argument to internal routine

/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
Timed out on t5_2ff
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/local/lib/python2.6/dist-packages/Pylons-0.9.6.2-py2.6.egg/pylons/middleware.py:11: DeprecationWarning: The webhelpers.rails package is deprecated.
- Please begin migrating to the new helpers in webhelpers.html,
  webhelpers.text, webhelpers.number, etc.  
- Import url_for() directly from routes, and redirect_to() from
  pylons.controllers.util (if using Pylons) or from routes.
- All Javascript support has been deprecated.  You can write link_to_remote()
  yourself or use one of the third-party Javascript libraries.
  from webhelpers.rails.asset_tag import javascript_path
/home/szimpatikus/reddit/r2/r2/lib/translation.py:32: DeprecationWarning: the md5 module is deprecated; use hashlib instead
  from md5 import md5
/home/szimpatikus/reddit/r2/r2/lib/manager/tp_manager.py:22: DeprecationWarning: the sha module is deprecated; use the hashlib module instead
  import pylons, sha
reddit app a.v01.lanten.hu:22470 started d251ba7 at 2011-04-15 11:29:22.716201
/usr/local/lib/python2.6/dist-packages/Pylons-0.9.6.2-py2.6.egg/pylons/middleware.py:11: DeprecationWarning: The webhelpers.rails package is deprecated.
- Please begin migrating to the new helpers in webhelpers.html,
  webhelpers.text, webhelpers.number, etc.  
- Import url_for() directly from routes, and redirect_to() from
  pylons.controllers.util (if using Pylons) or from routes.
- All Javascript support has been deprecated.  You can write link_to_remote()
  yourself or use one of the third-party Javascript libraries.
  from webhelpers.rails.asset_tag import javascript_path
/home/szimpatikus/reddit/r2/r2/lib/translation.py:32: DeprecationWarning: the md5 module is deprecated; use hashlib instead
  from md5 import md5
/home/szimpatikus/reddit/r2/r2/lib/manager/tp_manager.py:22: DeprecationWarning: the sha module is deprecated; use the hashlib module instead
  import pylons, sha
reddit app a.v01.lanten.hu:23006 started d251ba7 at 2011-04-15 11:30:21.855022
/usr/bin/env: optipng: No such file or directory
Warning: g.media_domain == g.domain. This may give untrusted content access to user cookies
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/local/lib/python2.6/dist-packages/Pylons-0.9.6.2-py2.6.egg/pylons/middleware.py:11: DeprecationWarning: The webhelpers.rails package is deprecated.
- Please begin migrating to the new helpers in webhelpers.html,
  webhelpers.text, webhelpers.number, etc.  
- Import url_for() directly from routes, and redirect_to() from
  pylons.controllers.util (if using Pylons) or from routes.
- All Javascript support has been deprecated.  You can write link_to_remote()
  yourself or use one of the third-party Javascript libraries.
  from webhelpers.rails.asset_tag import javascript_path
/home/szimpatikus/reddit/r2/r2/lib/translation.py:32: DeprecationWarning: the md5 module is deprecated; use hashlib instead
  from md5 import md5
/home/szimpatikus/reddit/r2/r2/lib/manager/tp_manager.py:22: DeprecationWarning: the sha module is deprecated; use the hashlib module instead
  import pylons, sha
reddit app a.v01.lanten.hu:25104 started d251ba7 at 2011-04-15 11:43:17.251516
Warning: g.media_domain == g.domain. This may give untrusted content access to user cookies
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
oEmbed call (http://api.embed.ly/v1/api/oembed?url=http%3A%2F%2Fwww.google.com%2Fwebfonts%3Fsubset%3Dlatin%26sort%3Dpop&maxwidth=600&format=json) failed to return content for http://www.google.com/webfonts?subset=latin&sort=pop
oEmbed call (http://api.embed.ly/v1/api/oembed?url=http%3A%2F%2Fwww.google.com%2Fwebfonts%3Fsubset%3Dlatin%26sort%3Dpop&maxwidth=600&format=json) failed to return content for http://www.google.com/webfonts?subset=latin&sort=pop
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
Error fetching t5_2g1
Traceback (most recent call last):
  File "<string>", line 110, in process_link
  File "/home/szimpatikus/reddit/r2/r2/lib/utils/utils.py", line 1167, in __call__
    result = self.function(*args)
  File "<string>", line 106, in _process_link
  File "<string>", line 94, in set_media
  File "<string>", line 75, in update_link
  File "/home/szimpatikus/reddit/r2/r2/lib/db/thing.py", line 241, in _commit
    self._set_data(self._type_id, self._id, **data_props)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 572, in set_thing_data
    return set_data(table, type_id, thing_id, **vals)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 529, in set_data
    i.execute(*inserts)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/sql/expression.py", line 1122, in execute
    return e._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1209, in _execute_clauseelement
    return connection._execute_clauseelement(elem, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 874, in _execute_clauseelement
    return self.__execute_context(context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 894, in __execute_context
    self._cursor_executemany(context.cursor, context.statement, context.parameters, context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 960, in _cursor_executemany
    self._handle_dbapi_exception(e, statement, parameters, cursor, context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 931, in _handle_dbapi_exception
    raise exc.DBAPIError.instance(statement, parameters, e, connection_invalidated=is_disconnect)
DataError: (DataError) invalid byte sequence for encoding "UTF8": 0xe17a7a
HINT:  This error can also happen if the byte sequence does not match the encoding expected by the server, which is controlled by "client_encoding".
 'INSERT INTO reddit_data_link (thing_id, key, value, kind) VALUES (%(thing_id)s, %(key)s, %(value)s, %(kind)s)' [{'kind': 'bool', 'thing_id': 3169L, 'key': 'has_thumbnail', 'value': 't'}, {'kind': 'pickle', 'thing_id': 3169L, 'key': 'media_object', 'value': '(dp1\nS\'oembed\'\np2\n(dp3\nS\'provider_url\'\np4\nS\'http://www.ted.com\'\np5\nsS\'description\'\np6\nVTED Talks Brian Greene fizikus elmagyar\xe1zza a szuperh\xfar-elm\xe9letet, azt a feltev\xe9st hogy apr\xf3, 11 dimenzi\xf3ban vibr\xe1l\xf3 energiah\xfarok alkotj\xe1k az \xf6sszes r\xe9szecsk\xe9t \xe9s er\\u0151t a Vil\xe1gegyetemben.\np7\nsS\'title\'\np8\nVBrian Greene a h\xfarelm\xe9letr\\u0151l | Video on TED.com\np9\nsS\'type\'\np10\nS\'video\'\np11\nsS\'thumbnail_width\'\np12\nI389\nsS\'height\'\np13\nI326\nsS\'width\'\np14\nI334\nsS\'html\'\np15\nS\'<object width="334" height="326"><param name="movie" value="http://video.ted.com/assets/player/swf/EmbedPlayer.swf"></param><param name="allowFullScreen" value="true"></param><param name="allowScriptAccess" value="always"></param><param name="wmode" value="transparent"></param><param name="bgColor" value="#ffffff"></param><param name="flashvars" value="vu=http://video.ted.com/talks/dynamic/BrianGreene_2005-medium.flv&su=http://images.ted.com/images/ted/tedindex/embed-posters/BrianGreene-2005.embed_thumbnail.jpg&vw=320&vh=240&ap=0&ti=251&lang=hun&introDuration=15330&adDuration=4000&postAdDuration=830&adKeys=talk=brian_greene_on_string_theory;year=2005;theme=peering_into_space;theme=unconventional_explanations;event=Unconventional+Explanations;tag=Science;tag=physics;tag=universe;&preAdTag=tconf.ted/embed;tile=1;sz=512x288;"></param><embed src="http://video.ted.com/assets/player/swf/EmbedPlayer.swf" pluginspace="http://www.macromedia.com/go/getflashplayer" type="application/x-shockwave-flash" wmode="transparent" bgcolor="#ffffff" width="334" height="326" allowfullscreen="true" allowscriptaccess="always" flashvars="vu=http://video.ted.com/talks/dynamic/BrianGreene_2005-medium.flv&su=http://images.ted.com/images/ted/tedindex/embed-posters/BrianGreene-2005.embed_thumbnail.jpg&vw=320&vh=240&ap=0&ti=251&lang=hun&introDuration=15330&adDuration=4000&postAdDuration=830&adKeys=talk=brian_greene_on_string_theory;year=2005;theme=peering_into_space;theme=unconventional_explanations;event=Unconventional+Explanations;tag=Science;tag=physics;tag=universe;"></embed></object>\'\np16\nsS\'version\'\np17\nS\'1.0\'\np18\nsS\'provider_name\'\np19\nS\'TED\'\np20\nsS\'thumbnail_url\'\np21\nS\'http://images.ted.com/images/ted/174_389x292.jpg\'\np22\nsS\'thumbnail_height\'\np23\nI291\nssS\'type\'\np24\nS\'ted.com\'\np25\ns.'}]

/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
Error fetching t5_2g2
Traceback (most recent call last):
  File "<string>", line 110, in process_link
  File "/home/szimpatikus/reddit/r2/r2/lib/utils/utils.py", line 1167, in __call__
    result = self.function(*args)
  File "<string>", line 106, in _process_link
  File "<string>", line 94, in set_media
  File "<string>", line 75, in update_link
  File "/home/szimpatikus/reddit/r2/r2/lib/db/thing.py", line 241, in _commit
    self._set_data(self._type_id, self._id, **data_props)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 572, in set_thing_data
    return set_data(table, type_id, thing_id, **vals)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 529, in set_data
    i.execute(*inserts)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/sql/expression.py", line 1122, in execute
    return e._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1209, in _execute_clauseelement
    return connection._execute_clauseelement(elem, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 874, in _execute_clauseelement
    return self.__execute_context(context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 894, in __execute_context
    self._cursor_executemany(context.cursor, context.statement, context.parameters, context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 960, in _cursor_executemany
    self._handle_dbapi_exception(e, statement, parameters, cursor, context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 931, in _handle_dbapi_exception
    raise exc.DBAPIError.instance(statement, parameters, e, connection_invalidated=is_disconnect)
DataError: (DataError) invalid byte sequence for encoding "UTF8": 0xf3647a2c
HINT:  This error can also happen if the byte sequence does not match the encoding expected by the server, which is controlled by "client_encoding".
 'INSERT INTO reddit_data_link (thing_id, key, value, kind) VALUES (%(thing_id)s, %(key)s, %(value)s, %(kind)s)' [{'kind': 'bool', 'thing_id': 3170L, 'key': 'has_thumbnail', 'value': 't'}, {'kind': 'pickle', 'thing_id': 3170L, 'key': 'media_object', 'value': '(dp1\nS\'oembed\'\np2\n(dp3\nS\'provider_url\'\np4\nS\'http://www.youtube.com/\'\np5\nsS\'description\'\np6\nVWe exist despite the sober formality of great concert halls, despite the boredom of classical musicians\' life, despite fanatic lovers of classical music, despite fans of rock, rap or pop who are afraid of classical music. We treat our Muse with a humorous irony and we\'re sure, she will have nothing against it!"\r\\u000a\r\\u000aThe names that make the moz\'ART Group are: Filip Jaslar - first violin; Micha\\u0142 Sikorski - second violin; Pawe\\u0142 Kowaluk - viola; Bolek B\\u0142aszczyk - cello. They are all well educated instrumentalists who graduated from prestigious Academies of Music in Warsaw and \\u0141\xf3dz, but they decided to play classical music in a humorous way. They created a worldwide unique musical cabaret, where the music, not the words are the source of joy and laugh. \r\\u000a\r\\u000aTake a classical composition as canvas, analyze its structure and theme, surround it with musical associations, brilliant, unbelievable ideas and while listening to the final product the listener is constantly surprised, amazed, laughing and moved to tears. \r\\u000a\r\\u000aThe musicians of the Moz\'ART GROUP have been playing together since 1995. At the beginning, they presented short musical jokes on Canal Plus Television. They gave their first debut in 1997 at the PAKA - competition of young Polish cabarets in Krakow. In the same year, they presented their first cabaret program entitled "Mozart\'s Still Alive" and since then, they have given concerts in Poland, Japan, China, the United States, Belgium, France and Switzerland. \r\\u000a\r\\u000aThe Moz\'ART GROUP is a frequent guest on Polish television including international galas and concerts. The Quartet received prestigious prizes like the Grand Prix of the XVIII Festival of Satire and Comedy in Lidzbark; "Pingwin z brazu" (the Bronze Penguin) - the prize given by the cabaret community of Zielona Gora, an informal capitol of Polish cabaret. The Moz\'AR Group performes together with the world-famous comic mime from Poland, Irek Krosny. \r\\u000aOne of the most important performances was a concert with Bobby McFerrin at the Congress Concert Hall in Warsaw. \r\\u000a\r\\u000aAnother show of the Group, in which they took Vivaldi as their patron, was entitled "The Four Seasons \xe0 la Moz\'ART GROUP". Similar subtitle has their latest CD "Creatures" where each of the "seasons" begins the same as Vivaldi\'s compositions. Later on, however, it runs freely in the direction of various world songs about spring, summer, autumn and winter. This show provides the listener with over 70 minutes of a sophisticated, perfectly arranged, sublime fun!\np7\nsS\'title\'\np8\nS\'MozART GROUP www.mozartgroup.org\'\np9\nsS\'url\'\np10\nS\'http://www.youtube.com/watch?v=SLYgVbVRoqk\'\np11\nsS\'type\'\np12\nS\'video\'\np13\nsS\'author_name\'\np14\nS\'artegemini\'\np15\nsS\'height\'\np16\nI363\nsS\'width\'\np17\nI600\nsS\'html\'\np18\nS\'<object width="600" height="363"><param name="movie" value="http://www.youtube.com/v/SLYgVbVRoqk?version=3"></param><param name="allowFullScreen" value="true"></param><param name="allowscriptaccess" value="always"></param><embed src="http://www.youtube.com/v/SLYgVbVRoqk?version=3" type="application/x-shockwave-flash" width="600" height="363" allowscriptaccess="always" allowfullscreen="true"></embed></object>\'\np19\nsS\'thumbnail_width\'\np20\nI480\nsS\'version\'\np21\nS\'1.0\'\np22\nsS\'provider_name\'\np23\nS\'YouTube\'\np24\nsS\'thumbnail_url\'\np25\nS\'http://i4.ytimg.com/vi/SLYgVbVRoqk/hqdefault.jpg\'\np26\nsS\'thumbnail_height\'\np27\nI360\nsS\'author_url\'\np28\nS\'http://www.youtube.com/user/artegemini\'\np29\nssS\'type\'\np30\nS\'youtube.com\'\np31\ns.'}]

/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
Error fetching t5_2g3
Traceback (most recent call last):
  File "<string>", line 110, in process_link
  File "/home/szimpatikus/reddit/r2/r2/lib/utils/utils.py", line 1167, in __call__
    result = self.function(*args)
  File "<string>", line 106, in _process_link
  File "<string>", line 94, in set_media
  File "<string>", line 75, in update_link
  File "/home/szimpatikus/reddit/r2/r2/lib/db/thing.py", line 241, in _commit
    self._set_data(self._type_id, self._id, **data_props)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 572, in set_thing_data
    return set_data(table, type_id, thing_id, **vals)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 511, in set_data
    keys = [x.key for x in s.execute().fetchall()]
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/sql/expression.py", line 1122, in execute
    return e._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1209, in _execute_clauseelement
    return connection._execute_clauseelement(elem, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 874, in _execute_clauseelement
    return self.__execute_context(context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 896, in __execute_context
    self._cursor_execute(context.cursor, context.statement, context.parameters[0], context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 948, in _cursor_execute
    self.dialect.do_execute(cursor, statement, parameters, context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/default.py", line 122, in do_execute
    cursor.execute(statement, parameters)
SystemError: null argument to internal routine

/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
Error fetching t5_2gh
Traceback (most recent call last):
  File "<string>", line 110, in process_link
  File "/home/szimpatikus/reddit/r2/r2/lib/utils/utils.py", line 1167, in __call__
    result = self.function(*args)
  File "<string>", line 106, in _process_link
  File "<string>", line 94, in set_media
  File "<string>", line 75, in update_link
  File "/home/szimpatikus/reddit/r2/r2/lib/db/thing.py", line 241, in _commit
    self._set_data(self._type_id, self._id, **data_props)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 572, in set_thing_data
    return set_data(table, type_id, thing_id, **vals)
  File "/home/szimpatikus/reddit/r2/r2/lib/db/tdb_sql.py", line 529, in set_data
    i.execute(*inserts)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/sql/expression.py", line 1122, in execute
    return e._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 1209, in _execute_clauseelement
    return connection._execute_clauseelement(elem, multiparams, params)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 874, in _execute_clauseelement
    return self.__execute_context(context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 894, in __execute_context
    self._cursor_executemany(context.cursor, context.statement, context.parameters, context=context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 960, in _cursor_executemany
    self._handle_dbapi_exception(e, statement, parameters, cursor, context)
  File "/usr/local/lib/python2.6/dist-packages/SQLAlchemy-0.5.3-py2.6.egg/sqlalchemy/engine/base.py", line 931, in _handle_dbapi_exception
    raise exc.DBAPIError.instance(statement, parameters, e, connection_invalidated=is_disconnect)
DataError: (DataError) invalid byte sequence for encoding "UTF8": 0xe96c22
HINT:  This error can also happen if the byte sequence does not match the encoding expected by the server, which is controlled by "client_encoding".
 'INSERT INTO reddit_data_link (thing_id, key, value, kind) VALUES (%(thing_id)s, %(key)s, %(value)s, %(kind)s)' [{'kind': 'bool', 'thing_id': 3185L, 'key': 'has_thumbnail', 'value': 't'}, {'kind': 'pickle', 'thing_id': 3185L, 'key': 'media_object', 'value': '(dp1\nS\'oembed\'\np2\n(dp3\nS\'provider_url\'\np4\nS\'http://www.youtube.com/\'\np5\nsS\'description\'\np6\nV"El\\u0151zenekarom sem lehetn\xe9l" album t\xf6lthet\\u0151 ingyenesen: http://mcpitaramos.hu/\r\\u000afell\xe9p\xe9sszervez\xe9s : 06-30-65-67-648\r\\u000aCsatlakozz: http://www.facebook.com/MCPITARAMOSOFFICIAL\r\\u000a\r\\u000arendezte: L\\u0151rincz Attila ( B\xfav\xe1rm\xe9dia) \r\\u000af\xe9nyk\xe9pezte, v\xe1gta: Tuli Gergely "Pixi" ( VariArt Studio)\r\\u000aFELL\xc9P\xc9SSZERVEZ\xc9S: 06-30-65-67-648!Okt\xf3ber 16.-\xe1n az MR2 Pet\\u0151fi r\xe1di\xf3b\xf3l is j\xf3l ismert MC Pita Ramos 2. lemeze     El\\u0151zenekarom sem lehetn\xe9l c\xedmmel.\r\\u000aAz album tartalmaz majd az el\\u0151z\\u0151 Reggaesztr\xe1ci\xf3 albumr\xf3l is 6 b\xf3nusz tracket, hogy j\xf3 min\\u0151s\xe9gben legyen meg nektek! Az album ez\xe1ltal nem fog t\xf6bbe ker\xfclni!S\\u0151t INGYENESEN LEHET MAJD T\xd6LTENI!!Aki k\xe9zzelfoghat\xf3 p\xe9ld\xe1nyt szeretne az rendelhet majd a k\xe9s\\u0151bbiekben.\r\\u000aA Tracklista:\r\\u000a\r\\u000a1. Aranylemez pedig Pet\\u0151fi is... ( skit)\r\\u000a2. MC Pita Ramos - Told fel a hangot!  ( A Lad\xe1nybene 27 judah riddim album\xe1n is megtal\xe1lhat\xf3)\r\\u000a3. MC Pita Ramos - Szeptember v\xe9g\xe9n ( hard version, Pet\\u0151fi verfeldolgoz\xe1s)\r\\u000a4. MC Pita Ramos - Falu v\xe9g\xe9n kurta kocsma k\xf6zr.: D\xf3zi ( Pet\\u0151fi versfeldolgoz\xe1s)\r\\u000a5. MC Pita Ramos - Hungarian Ragga\r\\u000a6. MC Pita Ramos - A Rem\xe9nyhez (Csokonai versfeldolgoz\xe1s)\r\\u000a7. MC Pita Ramos - Szeptember v\xe9g\xe9n (soft love version, Pet\\u0151fi versfeldolgoz\xe1s)\r\\u000a8. Mocskos celebek (skit)\r\\u000a9. MC Pita Ramos - El\\u0151zenekarom sem lehetn\xe9l\r\\u000a10. MC Pita Ramos - F\xf6ldre sz\xe1llt angyal (A f\xe1k is siratj\xe1k, bootleg)\r\\u000a11. MC Pita Ramos - Smokin k\xf6zr.: GerImpro (beatbox)\r\\u000a12. D\xf3zi - V\xe1rj m\xedg felkel majd a nap k\xf6zr.: MC Pita Ramos (bootleg)\r\\u000a13. MC Pita Ramos - Jimmy dala (Jesolo)\r\\u000a14. MC Pita Ramos - I wanna\r\\u000a15. MC Pita Ramos - Sz\xe1lln\xe9k ha tudn\xe9k\r\\u000a16. MC Pita Ramos - A l\xe1bamon alig alig\r\\u000a17. MC Pita Ramos - Ketten\r\\u000a18. MC Pita Ramos - \xdagy keverem\r\\u000a19. MC Pita Ramos - Mel\xf3dia\r\\u000a20. MC Pita Ramos - Tavaszi \xe9bred\xe9s\r\\u000a21. K\xf6sz\xf6n\xf6m (skit)\r\\u000a\r\\u000a\r\\u000aK\xfcl\xf6n k\xf6sz\xf6net:\r\\u000aA SZENNAI FALUM\xdaZEUMNAK, A MISZLANG CSAL\xc1DNAK, A L\\u0150RINCZ CSAL\xc1DNAK, HARKA BETTINEK, DOGNAK, HAWKINS-nak, VAJDA \xc1KOSNAK, A N\xc9PT\xc1NCOSOKNAK, ZS\xc9NEK, \xe9s sok mindenkinek....\np7\nsS\'title\'\np8\nVMC Pita Ramos km. D\xf3zi - Falu v\xe9g\xe9n kurta kocsma...  [OFFICIAL MUSIC VIDEO]\np9\nsS\'url\'\np10\nS\'http://www.youtube.com/watch?v=yqwujy9EaIM\'\np11\nsS\'type\'\np12\nS\'video\'\np13\nsS\'author_name\'\np14\nS\'pitaramos2\'\np15\nsS\'height\'\np16\nI363\nsS\'width\'\np17\nI600\nsS\'html\'\np18\nS\'<object width="600" height="363"><param name="movie" value="http://www.youtube.com/v/yqwujy9EaIM?version=3"></param><param name="allowFullScreen" value="true"></param><param name="allowscriptaccess" value="always"></param><embed src="http://www.youtube.com/v/yqwujy9EaIM?version=3" type="application/x-shockwave-flash" width="600" height="363" allowscriptaccess="always" allowfullscreen="true"></embed></object>\'\np19\nsS\'thumbnail_width\'\np20\nI480\nsS\'version\'\np21\nS\'1.0\'\np22\nsS\'provider_name\'\np23\nS\'YouTube\'\np24\nsS\'thumbnail_url\'\np25\nS\'http://i2.ytimg.com/vi/yqwujy9EaIM/hqdefault.jpg\'\np26\nsS\'thumbnail_height\'\np27\nI360\nsS\'author_url\'\np28\nS\'http://www.youtube.com/user/pitaramos2\'\np29\nssS\'type\'\np30\nS\'youtube.com\'\np31\ns.'}]

/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
/usr/bin/env: optipng: No such file or directory
scraper_q: 1 item 
